packages/universal-router-sdk/src/SwapRouterPancake.ts

```diff
@@ -0,0 +1,72 @@
+import { TradeType } from '@pancakeswap/sdk'
+import { SmartRouter, SmartRouterTrade } from '@pancakeswap/smart-router/evm'
+import { MethodParameters } from '@pancakeswap/v3-sdk'
+import invariant from 'tiny-invariant'
+import { encodeFunctionData, toHex, Hex } from 'viem'
+import { PancakeSwapTrade } from './entities/protocols/pancakeswap'
+import { encodePermit } from './utils/inputTokens'
+import { RoutePlanner } from './utils/routerCommands'
+import { PancakeSwapOptions, SwapRouterConfig } from './utils/types'
+import { UniversalRouterABI } from './abis/UniversalRouter'
+
+export abstract class PancakeUniversalSwapRouter {
+  /**
+   * Produces the on-chain method name to call and the hex encoded parameters to pass as arguments for a given trade.
+   * @param trades to produce call parameters for
+   * @param options options for the call parameters
+   */
+  public static swapERC20CallParameters(
+    trades: SmartRouterTrade<TradeType>,
```

Comment:
I think we accept multiple trades to be encoded at the same time? That's why we name it in plural

Reply:
yes we can accept trades theres below this which which does that chexk